cranelift/codegen/meta/src/gen_inst.rs

```diff
@@ -996,7 +996,7 @@ fn gen_format_constructor(format: &InstructionFormat, fmt: &mut Formatter) {
         fmtln!(fmt, "};");
 
         if imms_need_sign_extension {
-            fmtln!(fmt, "data.sign_extend_immediates(ctrl_typevar);");
+            fmtln!(fmt, "data.mask_immediates(ctrl_typevar);");
```

Comment:
Can we rename the boolean on this if-block (`imms_need_sign_extension`) as well to something that fits the new scheme?